src/lib/viewers/BaseViewer.js

```diff
@@ -1000,6 +1006,30 @@ class BaseViewer extends EventEmitter {
         return AnnotationMode.NONE;
     }
 
+    /**
+     * Updates localStorage with the state of the ftux cursor toggle
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    handleFtuxCursorToggle(key) {
+        if (!this.cache.get(key)) {
+            this.cache.set(key, true, true);
+        } else if (this.cache.get(key) && this.containerEl) {
+            switch (key) {
```

Comment:
Could we have a map of `key` -> `classname` so we could avoid this switch statement?